packages/ember-metal/lib/property_set.js

```diff
@@ -67,7 +67,7 @@ export function set(obj, keyName, value, tolerant) {
   }
 
   Ember.assert("You need to provide an object and key to `set`.", !!obj && keyName !== undefined);
-  Ember.assert('calling set on destroyed object', !obj.isDestroyed);
+  Ember.assert("calling set('" + keyName + "') on a destroyed object", !obj.isDestroyed);
```

Comment:
Hrmm... it looks like I'm getting

![screen shot 2015-06-09 at 5 44 43 pm](https://cloud.githubusercontent.com/assets/1817/8072777/e1012a4c-0ecf-11e5-80da-0e307a1e975f.png)

When I use:

```js
"') on a destroyed object (" + obj + ")"
```

And:

![screen shot 2015-06-09 at 5 48 20 pm](https://cloud.githubusercontent.com/assets/1817/8072780/f216844e-0ecf-11e5-86b0-4e81230989e6.png)

When I use:

```js
"') on a destroyed object (" + obj.toString() + ")"
```